cycle_time: 0.001                           # s
path_cache_size: 16
trajectory_cache_size: 16
trajectory_fifo_size: 30
duration_of_segment_in_trajectory_fifo: 0.01
stable_with_fine:
    cycle: 5            # 判断位置稳定时，位置误差连续小于门限值的次数
    threshold: 0.1      # mm
time_out_cycle:
    disable_to_standby: 2000
    standby_to_disable: 2000
    standby_to_auto: 150
    auto_to_standby: 250
    manual_to_standby: 250
    trajectory_flow: 50
    servo_update: 50

cartesian_vel_limit:
    min: 0.1
    max: 4000.0

simulator:
    enable: false
    cycle_per_publish: 25

cartesian:
    velocity:
        min: 1.0                            # mm/s
        max: 4000.0                         # mm/s
        default: 500.0                      # mm/s
        reference: 1000.0                   # mm/s
    acceleration:
        min: 1.0                            # mm/s^2
        max: 32000.0                        # mm/s^2
        default: 7000.0                     # mm/s^2
        reference: 4000.0                   # mm/s^2
    orientation:
        omega_reference: 0.2                # rad/s
        alpha_reference: 1.0                # rad/s^2
        linear_polation_threshold: 0.1      # rad

manual:
    step_length:
        joint: 0.1                          # rad
        position: 1.0                       # mm
        orientation: 0.01                   # rad

joint_tracking_accuracy: [0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01, 0.01]

calibrator:
    normal_offset_threshold: [10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0]
    lost_offset_threshold: [10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0]


#END
